src/main/java/fr/insalyon/creatis/gasw/plugin/listener/healing/execution/CommandState.java

```diff
@@ -168,7 +176,7 @@ private void doHealing(List<Job> jobs, List<Job> failedJobs,
                 }
             }
             if (jobs.size() < HealingConfiguration.getInstance().getMaxReplicas()
-                    && failedJobs.size() < GaswConfiguration.getInstance().getDefaultRetryCount()
+                    && (failedJobs.size() - 1) < GaswConfiguration.getInstance().getDefaultRetryCount()
```

Comment:
Not sure about that. If we have already 5 failed jobs (and 1 running), this will authorize a new replication leading to 7 jobs for the invocation. I thought of 6 (or 5 retries) as the max

Reply:
As discussed, with the ( size() - 1 ) implementation, we also allow for the replication of the last running job.